package.json

```diff
@@ -34,5 +34,9 @@
     "serverless-stack-termination-protection": "^1.0.4",
     "typescript": "^4.0.5",
     "yargs": "^16.1.1"
+  },
+  "dependencies": {
+    "serverless-deployment-bucket": "^1.5.1",
+    "serverless-s3-remover": "^0.6.0"
```

Comment:
I previously commented that this doesn't seem necessary (the remove plugin).  I didn't see any reason noted as to why it's here.  Any reason?  I'd ditch it, and any configuration you have for it.  Our destroy workflow is not reliant on serverless, including it's remover stuff.